### PR TITLE
Fix#1338 Navigation menu now appears for mobile users

### LINKED ISSUE #1338: Navigation menu is not coming on mobile after login.
Navigation menu icon is not coming up on mobile for logged in user.

### PR DESCRIPTION
**Pull request by Obtin**
Fix #1338 @aka-jain @RishabhJain2018 please review it and suggest me the changes. Thank you

### CODE REVIEW

**Comment by aka-jain** [comment]
@Obtin please attach a screeenshot here.

**Comment by Obtin** [comment]
@aka-jain  I have already posted them here https://github.com/Cloud-CV/EvalAI/issues/1338

**Comment by Obtin** [comment]
image
image

File: frontend/src/views/web/partials/sim-header.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
-<div class="sim-header ev-fixed white ">
-    <nav class="transparent ev-z-depth-5">
-        <div class="nav-wrapper padding-sm-lr-header">
-            <a href="#" data-activates="sidebarToggle" class="button-collapse text-med-gray hide-on-med-and-down"><i class="fa fa-bars"></i></a>
+div class="sim-header padding-lr-header">
+    <nav>
+        <div class="nav-wrapper">
+            <a href="#" data-activates="mobile-demo" class="button-collapse sim-header-link"><i class="fa fa-bars"></i></a>
```
Comment (aka-jain) [inline frontend/src/views/web/partials/sim-header.html]: @Obtin replace '#' with 'javascript:void(0)' here

File: frontend/src/css/modules/landing.scss [hunk 2]
```diff
@@ -107,11 +107,50 @@ section.bg-section {
...
-    z-index: 1001;
-    width: 100%;
+  font-size: 18px;
+  top: 0;
+  left: 0;
+  z-index: 999;
+  width: 100%;
+  position: relative;
+}
+.sim-header nav {
```
Comment (aka-jain) [inline frontend/src/css/modules/landing.scss]: In scss, for nested classes use this syntax
.a{
  .b{
     .c{
        }
    }
}
or take help for other files. @Obtin

**Comment by Obtin** [comment]
@aka-jain done with the requested changes. Please let me know if I have to make any other changes too. Thank you.

File: frontend/src/css/modules/landing.scss [hunk 3]
```diff
@@ -110,8 +110,47 @@ section.bg-section {
...
+        top: 0;
+        left: 0;
+        padding: 0 100px;
+        transition: all 0.1s ease-in-out;
+        a.sim-header-link {
+	  color: #4d4d4d;
+	  font-weight: 400;
+	  transition: all 0.2s ease-in-out;
+	}
+        a.sim-header-link:hover {
```
Comment (aka-jain) [inline frontend/src/css/modules/landing.scss]: @Obtin you can make these nested too using &.
Like a{&.sim-header-link{}}

Comment (Obtin) [reply frontend/src/css/modules/landing.scss]: @aka-jain done.

File: frontend/src/css/modules/landing.scss [hunk 4]
```diff
@@ -110,8 +110,47 @@ section.bg-section {
...
+        top: 0;
+        left: 0;
+        padding: 0 100px;
+        transition: all 0.1s ease-in-out;
+        a.sim-header-link {
+	  color: #4d4d4d;
+	  font-weight: 400;
+	  transition: all 0.2s ease-in-out;
+	}
+        &:hover {
```
Comment (aka-jain) [inline frontend/src/css/modules/landing.scss]: Have you test this? Because I think this hover should come inside `a`

**Comment by aka-jain** [comment]
There are 2 extra files in your commits. Please remove them as they are of no use. @Obtin

**Comment by Obtin** [comment]
@aka-jain done

**Comment by RishabhJain2018** [comment]
@Obtin I think you missed something. Please check this screenshot!
timeline8

**Comment by Obtin** [comment]
@RishabhJain2018, I checked it and its working fine I guess.I verified it using Chrome dev tools.
devtools

File: frontend/src/views/web/partials/sim-header.html [hunk 5]
```diff
@@ -14,6 +14,24 @@
...
+        <li ng-show="isAuth"><a class="waves-effect waves-dark sim-header-link" ui-sref="web.dashboard" ui-sref-active="active"><i class="fa fa-thumb-tack"></i>Dashboard</a></li>
+        <li ng-show="isAuth"><a class="waves-effect waves-dark sim-header-link" ui-sref="web.challenge-main" ui-sref-active="active"><i class="fa fa-fire"></i>All Challenges</a></li>
+        <li ng-show="isAuth"><a class="waves-effect waves-dark sim-header-link" ui-sref="web.teams" ui-sref-active="active"><i class="fa fa-users"></i>Participant Teams</a></li>
+        <li ng-show="isAuth"><a class="waves-effect waves-dark sim-header-link" ui-sref="web.challenge-host-teams" ui-sref-active="active"><i class="fa fa-paper-plane"></i>Create Challenge</a></li>
+        <li ng-show="isAuth"><a class="waves-effect waves-dark sim-header-link" ui-sref="auth.signup" ui-sref-active="active"><i class="fa fa-user"></i>Hi {{web.name}}!</a></li>
+      </ul>
+
         </div>
     </nav>
 </div>
```
Comment (RishabhJain2018) [inline frontend/src/views/web/partials/sim-header.html]: @Obtin Yes, you are right, The code works perfectly! I want you to add the missing fields in the navigation of `Home` page. Please see the two screenshots.
1. Home Page Menu
timeline10
2. Dashboard Menu
timeline11

**Comment by RishabhJain2018** [comment]
Please let me know if I am not clear.

**Comment by Obtin** [comment]
@RishabhJain2018 
home page1

**Pull request merged**